tests/boilerplate_client/boilerplate_cmd.py

```diff
@@ -1,7 +1,9 @@
 import struct
-from typing import Tuple
+from typing import Tuple, Generator
+from contextlib import contextmanager
 
-from ledgercomm import Transport
+from ragger.backend.interface import BackendInterface, RAPDU
+from ragger.error import ExceptionRAPDU
 
 from boilerplate_client.boilerplate_cmd_builder import BoilerplateCommandBuilder, InsType
 from boilerplate_client.button import Button
```

Comment:
```suggestion
from boilerplate_client.button import Button
```
Not used, neither DeviceException below.

Reply:
Removed